lxd/patches.go

```diff
@@ -954,23 +1091,28 @@ func upgradeFromStorageTypeZfs(name string, d *Daemon, defaultPoolName string, d
 	for _, img := range images {
 		_, err := dbStoragePoolVolumeCreate(d.db, img, storagePoolVolumeTypeImage, poolID, volumeConfig)
 		if err != nil {
+			// In case we're on a LXD instance where a a partial
+			// upgrade took place, we should still proceed.
 			shared.LogWarnf("Could not insert a storage volume for image \"%s\".", img)
-			continue
```

Comment:
Same as above, should be fatal unless there's actually the right record in the DB.